api/document/management/commands/import_xml_doc.py

```diff
@@ -17,6 +18,14 @@ def fetch_policy(identifier: str):
         return Policy.objects.filter(omb_policy_id=identifier).first()
 
 
+def import_xml_doc(policy, xmlstream):
+    parsed_data = AkomaNtosoParser().parse(xmlstream)
+    cursor = DocCursor.new_tree('', policy=policy)
```

Comment:
Aggh I _really_ dislike this line.  Our `DocCursorSerializer` was built to serialize existing `DocCursor`s, not to create brand-new ones. So here I'm just "creating" a new `DocCursor` for no other reason than so our importer can eventually pluck out its `policy`. Ooooof.

Reply:
Would it make sense to override the serializer's `create()` method? We'd need to pass the policy as context, but I wonder if that's where folks would expect this sort of logic?

Reply:
Ah that's a great idea! I just filed #950 to address it.